apps/app/src/lib/auth/stackAuth.ts

```
@@ -1,9 +1,9 @@
+import { StackClientApp } from "@stackframe/react";
```

ESLint

[error] Resolve error: EACCES: permission denied, open '/XKtMQSwEnD'
    at Object.writeFileSync (node:fs:2409:20)
    at l (/home/jailuser/git/node_modules/.pnpm/get-tsconfig@4.10.1/node_modules/get-tsconfig/dist/index.cjs:7:13685)
    at createFilesMatcher (/home/jailuser/git/node_modules/.pnpm/get-tsconfig@4.10.1/node_modules/get-tsconfig/dist/index.cjs:7:14437)
    at Object.resolve (/home/jailuser/git/node_modules/.pnpm/eslint-import-resolver-typescript@4.4.4_eslint-plugin-import@2.32.0_eslint@9.33.0_jiti@2.5.1_/node_modules/eslint-import-resolver-typescript/lib/index.cjs:298:107)
    at withResolver (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.39.0_eslint@9.33.0_jiti@2.5.1__t_21ffdcf7f0f2c18e7b839026d32ed874/node_modules/eslint-module-utils/resolve.js:180:23)
    at fullResolve (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.39.0_eslint@9.33.0_jiti@2.5.1__t_21ffdcf7f0f2c18e7b839026d32ed874/node_modules/eslint-module-utils/resolve.js:201:22)
    at relative (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.39.0_eslint@9.33.0_jiti@2.5.1__t_21ffdcf7f0f2c18e7b839026d32ed874/node_modules/eslint-module-utils/resolve.js:217:10)
    at resolve (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.39.0_eslint@9.33.0_jiti@2.5.1__t_21ffdcf7f0f2c18e7b839026d32ed874/node_modules/eslint-module-utils/resolve.js:233:12)
    at ExportMapBuilder.get (/home/jailuser/git/node_modules/.pnpm/eslint-plugin-import@2.32.0_@typescript-eslint+parser@8.39.0_eslint@9.33.0_jiti@2.5.1___ae62935542874a0362555bdd9731fd8a/node_modules/eslint-plugin-import/lib/exportMap/builder.js:37:45)
    at processBodyStatement (/home/jailuser/git/node_modules/.pnpm/eslint-plugin-import@2.32.0_@typescript-eslint+parser@8.39.0_eslint@9.33.0_jiti@2.5.1___ae62935542874a0362555bdd9731fd8a/node_modules/eslint-plugin-import/lib/rules/namespace.js:12:38)

(import/namespace)